packages/turbo-repository/rust/src/lib.rs

```diff
@@ -182,6 +188,27 @@ impl Workspace {
         Ok(map)
     }
 
+    pub fn get_lockfile_contents(
+        &self,
+        changed_files: &HashSet<AnchoredSystemPathBuf>,
+        workspace_root: &AbsoluteSystemPath,
+        from_commit: &str,
+    ) -> LockfileContents {
+        let lockfile_name = self.graph.package_manager().lockfile_name();
+        changed_files
+            .contains(AnchoredSystemPath::new(&lockfile_name).unwrap())
+            .then(|| {
+                let git = SCM::new(workspace_root);
+                let anchored_path = workspace_root.join_component(lockfile_name);
+                git.previous_content(Some(from_commit), &anchored_path)
+                    .map(LockfileContents::Changed)
+                    .inspect_err(|e| debug!("{e}"))
+                    .ok()
+                    .unwrap_or(LockfileContents::UnknownChange)
+            })
+            .unwrap_or(LockfileContents::Unknown)
```

Comment:
I think this should be `LockfileContents::Unchanged` since the lockfile wasn't in the list of changed files
```suggestion
            .unwrap_or(LockfileContents::Unchanged)
```